web-components/src/components/tabs/TabPanel.ts

```diff
@@ -29,6 +31,9 @@ export namespace TabPanel {
       if (this.focusablePanel) {
         this.setAttribute("tabindex", "0");
       }
+      if(this.removeFocus){
```

Comment:
how would a developer know which focus will be removed?

Reply:
updated